docs/Actions/inject-data.md

```diff
@@ -0,0 +1,274 @@
+    "ActionType": "InjectData",
+    "Description": null,
+    "Condition": null,
+    "Parameters": {
+        "Data": [
+            {
+                "value": "ID15367",
+                "name": "ClientId"
+            }
+        ]
+    }
+}
+```
+
+### `3. Concatenate 2 tokens`
+
+​
+The action below extract "FirstName" and "LastName" from users database and concatenate them so that "FullName" can be used in the form. <br></br> [Import it](/docs/Actions/import-actions.md) into your application to see it in action.
+​
+
+```json
+{
+    "Title": "Execute Actions",
+    "ActionType": "ExecuteActions",
+    "Description": "Inject first user full name into form",
+    "Condition": null,
+    "Parameters": {
+        "ActionList": [
+            {
+                "Id": -1,
+                "$_uid": "action15911732046536029",
+                "Parameters": {
+                    "ConnectionString": "",
+                    "QueryTimeout": "",
+                    "SqlQuery": "SELECT FirstName, LastName FROM Users\nWHERE UserId = 1",
+                    "BindTokens": [],
+                    "OutputTokenName": "",
+                    "ExtractColumns": [
+                        {
+                            "value": "FirstName",
+                            "name": "FirstName"
+                        },
+                        {
+                            "value": "LastName",
+                            "name": "LastName"
+                        }
+                    ],
+                    "OnError": [],
+                    "ShowErrors": false
+                },
+                "ActionType": "RunSql",
+                "$_isOpen": false,
+                "$_isLoaded": true,
+                "$_isFocus": true,
+                "Definition": {
+                    "IsClientAction": false,
+                    "Settings": {
+                        "Group": "Data"
+                    }
+                }
+            },
+            {
+                "Id": -1,
+                "$_uid": "action15911732046533516",
+                "Parameters": {
+                    "Data": [
+                        {
+                            "value": "[FirstName] [LastName]",
+                            "name": "FullName"
+                        }
+                    ]
+                },
+                "ActionType": "InjectData",
+                "$_isOpen": false,
+                "$_isLoaded": true,
+                "$_isFocus": true,
+                "Definition": {
+                    "IsClientAction": false,
+                    "Settings": {
+                        "Group": "Context"
+                    }
+                }
+            }
+        ]
+    }
+}
+```
+
+### `4. Overwrite a token`
+
+​
+The action below creates a token with initial value "1" and overwrites it. The injected token will have value "2", which is the new one. <br></br> [Import it](/docs/Actions/import-actions.md) into your application to see it in action.
```

Comment:
Let's find a real-life example when we'd do this.

Reply:
Found one and added it